clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/data/Keys.groovy

```diff
@@ -98,8 +100,19 @@ class Keys {
     "${AWS}:${Namespace.LAUNCH_CONFIGS}:${account}:${region}:${launchConfigName}"
   }
 
-  static String getLoadBalancerKey(String loadBalancerName, String account, String region, String vpcId) {
-    "${AWS}:${Namespace.LOAD_BALANCERS}:${account}:${region}:${loadBalancerName}${vpcId ? ':' + vpcId : ''}"
+  static String getLoadBalancerKey(String loadBalancerName, String account, String region, String vpcId, String type) {
```

Comment:
ah yeah good call, I was initially trying to add just a 'type' attribute to the parsed key until I realized it clashed with the cloud provider type